docs/about/github_stars.py

```diff
@@ -0,0 +1,147 @@
+from bokeh.plotting import figure, show
```

Comment:
copyright header missing

Reply:
Fixed

Reply:
Our new header format should be:
```
# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Which reminds me--we should propagate this to the rest of the repository and say that that's our format.